lib/matplotlib/dates.py

```diff
@@ -1393,8 +1393,10 @@ def __init__(self, base=1, month=1, day=1, tz=None):
                          'hour':   0,
                          'minute': 0,
                          'second': 0,
-                         'tzinfo': tz
                          }
+        # Note that tzinfo does not work with pytz timezones, and
```

Comment:
This is moot if we drop py3.5?

Reply:
ah, this is targetted at 3.0.x so have to support 3.5 still.

Reply:
Yep...  Could milestone 3.1 if you prefer.